aepsych/utils.py

```diff
@@ -16,7 +16,7 @@
 
 from aepsych.config import Config
 
-def make_scaled_sobol(lb, ub, size, seed=None):
+def make_scaled_sobol(lb : Union[torch.Tensor, np.ndarray], ub : Union[torch.Tensor, np.ndarray], size: int, seed: Optional[int] = None) -> torch.Tensor:
     lb, ub, ndim = _process_bounds(lb, ub, None)
```

Comment:
array/tensor question

Reply:
It is true that `make_scaled_sobol` does not need this `Union[torch.Tensor, np.ndarray]`. However, `_process_bounds` and `promote_0d` methods still need to have this, as `_process_bounds` is used in almost the entire codebase. Fixing this will be the focus of #379 .